core/controllers/contributor_dashboard_admin_test.py

```diff
@@ -1320,7 +1321,7 @@ def test_get_translation_reviewer_stats_for_last_activity_filter(
     ) -> None:
         self.login(self.CONTRIBUTOR_EMAIL)
 
-        # Test with max_days_since_last_activity filter and pagination.
```

Comment:
Please read your PRs before sending them and get rid of the unnecessary or stale changes like this one

Reply:
Sorry @nikitaevg, fixed all such stale code.